generator-theia/src/extension/extension-package-generator.ts

```diff
@@ -55,6 +55,16 @@ export class ExtensionPackageGenerator extends AbstractGenerator {
     protected compileCommonPackage(): NodePackage {
         const keywords = new Set(this.model.pck.keywords || []);
         keywords.add(this.model.extensionConfig.extensionKeyword);
+        let buildCmd = "";
+        if (this.model.extensionConfig.internalizationSupport) {
+            const srcPath = this.model.extensionConfig.i18nSrcFolder + "/*.json";
+            const destPath = this.model.extensionConfig.i18nDestFolder;
+            // tslint:disable-next-line:max-line-length
+            buildCmd = "shx mkdir -p " + destPath + " && shx cp " + srcPath + " " + destPath + " &&  concurrently -n compile,lint -c blue,green \"yarn run compile\" \"yarn run lint\"";
```

Comment:
copying each time is not really nice:
- let's an introduce a convention for the extension developers that `i18n` folder should be next to `src`
- change the extension generator that it adds `i18n` folder to files property, similar to `lib` and `src`